checker/tests/nullness-checkcastelementtype/Issue1315.java

```diff
@@ -13,7 +13,7 @@ public class Issue1315 {
 
         @SuppressWarnings("unchecked")
         T test1(@Nullable Object p) {
-            // :: warning: (cast.unsafe)
+            // :: error: (cast.incompatible)
```

Comment:
`incompatible cast from "@Initialized @Nullable Object" to "T[ extends @Initialized @Nullable Object super @Initialized @NonNull Void]"`

How is it incompatible?

Reply:
fixed